src/@primer/gatsby-theme-doctocat/components/ImageBox.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import {Box} from '@primer/react'
+import { Caption } from '@primer/gatsby-theme-doctocat'
+
+function ImageBox({caption, children, props, paddingX = false}) {
```

Comment:
Non-blocking nitpick: I personally prefer to visually associate the caption with the image using whitespace instead of borders and matching background colors. It feels a little quieter and cleaner to me, but I realize that's just personal preference.

<img width="977" alt="Screenshot 2023-01-23 at 10 47 20 AM" src="https://user-images.githubusercontent.com/2313998/214083716-a16a9f67-6009-43eb-b6e2-fcc25a2f3303.png">

Reply:
This is how it was in figma, but I am with you as long as we keep the bg behind the img. They feel lost without any bg somehow. Will update 👍

Reply:
Preview: https://primer-32c707d434-26441320.drafts.github.io/guides/figma/contribute